frontend/src/components/FeatureSection.tsx

```diff
@@ -0,0 +1,55 @@
+import { Box, Heading, Flex, Text, Image, Button} from '@chakra-ui/react';
+import { QuestionIcon } from '@chakra-ui/icons';
+
+interface FeatureSectionProps {
+    headline:string
+    content:string
+    buttonLabel:string
+    buttonUrl:string
+}
+
+const FeatureSection = ({headline, content, buttonLabel, buttonUrl} : FeatureSectionProps) : JSX.Element => {
```

Comment:
A nicer way to type a component with TS is to import FC from react and do `const FeatureSection: FC<FeatureSectionProps> = ({ destructured props }) => {`

Reply:
And don't worry about typing the returned element as it will always be `JSX.Element` due to Babel transpiling